src/components/Chat/AnonymousInputMessage.tsx

```diff
@@ -120,4 +120,4 @@ const AnonymousInputMessage = ({onSendButtonClicked}: AnonymousInputMessageProps
   );
 };
 
-export default AnonymousInputMessage;
+export default React.memo(AnonymousInputMessage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Wrapping the component with `React.memo` is a good practice for performance optimization as it prevents unnecessary re-renders if the props haven't changed.
```diff
- export default AnonymousInputMessage;
+ export default React.memo(AnonymousInputMessage);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->